radius_server/Dockerfile.freeradius

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:18.04
```

Comment:
is there a reason not to just put etc/freeradius under radius_server/?

Reply:
No. I just thought having two etc folders might look ugly. Correcting.

Reply:
Done